index.js

```diff
@@ -0,0 +1,106 @@
+// import express (after npm install express)
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// Getting data from json files
+var packageData = require('./package.json');
+var categoryJson = require('./src/data/categories.json');
+var productJson = require('./src/data/products.json');
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+
+  var jsonOutputText = '{ "serverName": "' + packageData.name + '", "serverVersion": "' + packageData.version + '" }';
+  res.send(jsonOutputText);
+});
+
+// route for displaying all products
+app.get('/products/all', (req, res) => {
```

Comment:
Not related to this task

Reply:
I'm sorry, I pushed from wrong branch (which was for 3rd react task)